src/components/SlateEditor/plugins/keyPerformanceIndicator/KeyPerformanceIndicatorForm.tsx

```diff
@@ -0,0 +1,132 @@
+import { KeyPerformanceIndicatorEmbedData } from '@ndla/types-embed';
+import { spacing } from '@ndla/core';
+import { FieldProps, Formik } from 'formik';
+import { useCallback, useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import InlineImageSearch from '../../../../containers/ConceptPage/components/InlineImageSearch';
+import { supportedLanguages } from '../../../../i18n2';
+import FormikField from '../../../FormikField';
+import validateFormik, { RulesType } from '../../../formikValidationSchema';
+
+interface Props {
+  onSave: (data: KeyPerformanceIndicatorEmbedData) => void;
+  initialData: KeyPerformanceIndicatorEmbedData;
+  onCancel: () => void;
+}
+
+interface KeyPerformanceIndicatorFormValue {
+  resource: 'key-performance-indicator';
+  metaImageId: string;
+  language: string;
+  title: string;
+  subTitle: string;
+}
+
+const toInitialValues = (
+  initialData: KeyPerformanceIndicatorEmbedData,
+): KeyPerformanceIndicatorFormValue => ({
+  resource: initialData?.resource ?? 'key-performance-indicator',
+  metaImageId: initialData?.imageId ?? '',
+  title: initialData?.title ?? '',
+  subTitle: initialData?.subTitle ?? '',
+  language: initialData?.language ?? 'nb',
+});
+
+const rules: RulesType<KeyPerformanceIndicatorFormValue> = {
+  title: {
+    required: true,
+  },
+  subTitle: {
+    required: true,
+  },
+  language: {
+    required: true,
+  },
+};
+
+const StyledFormikField = styled(FormikField)`
+  margin: 0px;
+`;
+
+const inputStyle = css`
+  display: flex;
+  flex-direction: column;
+`;
+
+const ButtonContainer = styled.div`
+  display: flex;
+  justify-content: flex-end;
+  gap: ${spacing.small};
+`;
+
+const KeyPerformanceIndicatorForm = ({ onSave, initialData, onCancel }: Props) => {
+  const { t } = useTranslation();
+  const initialValues = useMemo(() => toInitialValues(initialData), [initialData]);
+  const initialErrors = useMemo(() => validateFormik(initialValues, rules, t), [initialValues, t]);
+
+  const onSubmit = useCallback(
+    (values: KeyPerformanceIndicatorFormValue) => {
+      const newData: KeyPerformanceIndicatorEmbedData = {
+        resource: 'key-performance-indicator',
+        imageId: values.metaImageId,
+        title: values.title,
+        subTitle: values.subTitle,
+        language: values.language,
+      };
+      onSave(newData);
+    },
+    [onSave],
+  );
+  return (
+    <Formik
+      initialValues={initialValues}
+      initialErrors={initialErrors}
+      onSubmit={onSubmit}
+      validateOnMount
+      validate={(values) => validateFormik(values, rules, t)}
+    >
+      {({ dirty, isValid, handleSubmit }) => (
+        <>
+          <StyledFormikField name="title" showError>
+            {({ field }: FieldProps) => (
+              <InputV2 customCss={inputStyle} label={t('form.name.title')} {...field} />
+            )}
+          </StyledFormikField>
+          <StyledFormikField name="subTitle" showError>
```

Comment:
Subtitle er et heilt fint ord. Treng ikkje casing her. Må gjerne også oppdatere backend om det skal endres.
```suggestion
          <StyledFormikField name="subtitle" showError>
```